client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -0,0 +1,135 @@
+				return 'is bad';
+			default:
+				return 'unknown';
+		}
+	};
+
+	const { good, needsImprovement } = metricsTresholds[ activeTab as keyof typeof metricsTresholds ];
+	const formatUnit = ( value: number ) => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return +( value / 1000 ).toFixed( 2 );
+		}
+
+		return value;
+	};
+
+	const displayUnit = () => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return 'seconds';
+		}
+
+		if ( [ 'inp' ].includes( activeTab ) ) {
+			return 'milliseconds';
+		}
+
+		return '';
+	};
+
+	return (
+		<div className="core-web-vitals-display">
+			<MetricTabBar activeTab={ activeTab } setActiveTab={ setActiveTab } { ...props } />
+			<div className="core-web-vitals-display__details">
+				<div className="core-web-vitals-display__description">
+					<span className="core-web-vitals-display__description-subheading">
+						{ translate( "Your site's %(displayName)s %(valuation)s", {
+							args: {
+								displayName: displayName.toLowerCase(),
+								valuation: displayValuation( valuation ),
+							},
+						} ) }
+					</span>
+					<MetricScale metricName={ activeTab } value={ value } valuation={ valuation } />
+					<div className="core-web-vitals-display__ranges">
+						<div className="range">
+							<StatusIndicator speed="good" />
+							<div className="range-description">
+								<div className="range-heading">{ translate( 'Fast ' ) }</div>
+								<div className="range-subheading">
+									{ translate( '0–%(to)s %(unit)s', {
+										args: { to: formatUnit( good ), unit: displayUnit() },
+									} ) }
+								</div>
+							</div>
+						</div>
+						<div className="range">
+							<StatusIndicator speed="needsImprovement" />
+							<div className="range-description">
+								<div className="range-heading">{ translate( 'Moderate' ) }</div>
+								<div className="range-subheading">
+									{ translate( '%(from)s–%(to)s %(unit)s', {
+										args: {
+											from: formatUnit( good ),
+											to: formatUnit( needsImprovement ),
+											unit: displayUnit(),
+										},
+									} ) }
+								</div>
+							</div>
+						</div>
+						<div className="range">
+							<StatusIndicator speed="bad" />
+							<div className="range-description">
+								<div className="range-heading">{ translate( 'Slow' ) }</div>
+								<div className="range-subheading">
+									{ translate( '%(from)s+ %(unit)s', {
+										args: {
+											from: formatUnit( needsImprovement ),
+											unit: displayUnit(),
+										},
+									} ) }
+								</div>
+							</div>
+						</div>
+					</div>
+					<span className="core-web-vitals-display__description-subheading">
+						{ translate( 'What is %(displayName)s? ', {
+							args: { displayName: displayName.toLowerCase() },
+						} ) }
+					</span>
+					<span className="core-web-vitals-display__description-aka">
+						{ translate( '(aka %(metricName)s)', {
+							args: { metricName },
```

Comment:
Same about making sure `metricsName` is translated.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/93411/commits/29f95b09e425e41053145bd30207ee7cc561c8b7